sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -6,15 +6,109 @@ export const ContentPlacementContainerStyles = css`
         contain: none;
         font-family: var(--body-font);
     }
-    .container {
-        display: flex;
-        flex-wrap: wrap;
+
+    .framework_container,
+    .community_container {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
         justify-content: center;
     }
+
+    .framework_ContentPlacement {
```

Comment:
Is there any way to do this with `nth-of-type` instead of adding a specific class to each item?

Reply:
Yes! I was having issues with it before, but it is working now. I will switch them all out and remove the individual classes.

Reply:
Yes! I was having issues with it before, but it is working now. I will switch them all out and remove the individual classes. It is still for some reason not recognizing when I try to pass it even or odd though. for example .framework_ContentPlacement:nth-of-type(even) will not work but 
        .framework_ContentPlacement:nth-of-type(2),
        .framework_ContentPlacement:nth-of-type(4),
        .framework_ContentPlacement:nth-of-type(6) 
will work.